docs/vtex-io/App-Guides/vtex-io-documentation-making-your-new-app-version-publicly-available/using-the-docs-builder-to-publish-app-documentation.md

```diff
@@ -3,7 +3,7 @@ title: "Using the Docs builder to publish app documentation"
 slug: "using-the-docs-builder-to-publish-app-documentation"
```

Comment:
Atualizar a slug e o nome do arquivo. Depois criar o redirect de `using-the-docs-builder-to-publish-app-documentation` para a nova rota.

@henriquessb usou o padrão `vtex-io-documentation-{nomeDoBuilder}-builder` nas [docs de builder que ele fez](https://github.com/vtexdocs/dev-portal-content/pull/1046/files). Acho que vale seguir para todas